tests/conftest.py

```diff
@@ -26,35 +24,108 @@
 
 @pytest.fixture
 def cache(tmpdir, scope='function'):
+    """Local cache folder."""
     return audeer.mkdir(audeer.path(tmpdir, 'cache'))
 
 
 @pytest.fixture
-def publish_db(tmpdir, scope='session', autouse=True):
-    r"""Publish a test database.
-
-    The database will use ``pytest.NAME`` as name
-    and ``pytest.VERSION`` as version.
-    It will be published to a file-system repository
-    in a tmp folder.
-    ``audb.config.REPOSITORIES`` will be adjusted
-    to point to the repository
-    the database was published to.
-
-    """
+def audb_cache(tmpdir, scope='session', autouse=True):
+    """Local audb cache folder."""
     cache = audeer.mkdir(audeer.path(tmpdir, 'audb-cache'))
     audb.config.CACHE_ROOT = cache
     audb.config.SHARED_CACHE = cache
 
-    db_path = audeer.mkdir(audeer.path(tmpdir, pytest.NAME))
+
+@pytest.fixture
+def repository(tmpdir, scope='session'):
+    """Local audb repository only visible inside the tests."""
+    host = audeer.mkdir(audeer.path(tmpdir, 'repo'))
+    repository = audb.Repository(
+        name='data-local',
+        host=host,
+        backend='file-system',
+    )
+    audb.config.REPOSITORIES = [repository]
+    return repository
+
+
+@pytest.fixture
+def minimal_db(
+        tmpdir,
+        repository,
+        scope='session',
+        autouse=True,
+):
+    r"""Publish and load a minimal database.
+
+    The name of the database will be ``minimal_db``.
+
+    The database has no schemes
+    and a single filewise table.
+
+    Further it contains a single file
+    with a length of 0.01 s.
+
+    """
+    name = 'minimal_db'
+
+    db_path = audeer.mkdir(audeer.path(tmpdir, name))
+
+    db = audformat.Database(
```

Comment:
I wonder whether the fixtures can be made modular? 
By this I mean that one fixture uses the other one as input. Probably one has to use the scope decorators appropriately. For me it would feel natural to use `minimal_db` as input to `medium_db`.

Reply:
I added another even smaller database, called `bare_db` and fixed a few related issue in the templates.

I added it also to see if it makes sense to pass the database on to other fixtures as you proposed above. I was not really convinced by it, at least for the databases we have at the moment as we would not really save any code by it and it also makes them harder to understand.

But I think in principle it should be possible to just pass on one smaller database to another database fixture where you can then extend that database and publish as another one.